internal/pkg/instrumentation/bpf/database/sql/probe.go

```diff
@@ -27,6 +29,9 @@ const (
 
 	// IncludeDBStatementEnvVar is the environment variable to opt-in for sql query inclusion in the trace.
 	IncludeDBStatementEnvVar = "OTEL_GO_AUTO_INCLUDE_DB_STATEMENT"
+
+	// IncludeDBOperationEnvVar is the environment variable to opt-in for sql query operation in the trace.
+	IncludeDBOperationEnvVar = "OTEL_GO_AUTO_INCLUDE_DB_OPERATION"
```

Comment:
Can we call this `OTEL_GO_AUTO_PARSE_DB_STATEMENT` instead? Give we are going to use the parsed semantics for more than just the operation it might help better convey the intention.

Reply:
done